metadata-protobuf/proto/WorkingGroups.proto

```diff
@@ -63,8 +63,14 @@ message ModeratePost {
   required string rationale = 2;
 }
 
+message VerifyValidator{
+  required uint64 memberId = 1;
+  required bool is_verified = 2;
+}
+
 message RemarkMetadataAction {
   oneof action {
     ModeratePost moderate_post = 1;
+    VerifyValidator verfy_validator = 2;
```

Comment:
This is correct but it should be on #4868 not here (because you'll need it for the tests)